test/builders/build/eth-api/libraries/viem/send-transaction.js

```diff
@@ -0,0 +1,66 @@
+import { assert } from 'chai';
```

Comment:
Can you please run prettier on this file? There is a config file (`.editorconfig`) so you should be able to open this in your editor and format the document based on the settings in the config file